Packs/Luminate/pack_metadata.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "Luminate",
     "description": "Enrich your report and Respond to incidents with Luminate",
-    "support": "partner",
-    "currentVersion": "1.0.0",
+    "support": "community",
```

Comment:
@marcellarichmond @mchasepan Why have we decided to change Aella Star Light to community and not XSOAR?

Reply:
Per @NeelRust